nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -360,14 +360,15 @@ in
       serviceConfig = let
         runOptions = ''--config ${configPath} ${optionalString (cfg.adapter != null) "--adapter ${cfg.adapter}"}'';
       in {
+        # Override the `ExecStart` line from upstream's systemd unit file by our own:
```

Comment:
@emilylange Because the code and existing comment are unclear to anybody who didn't write them. They explain how one can reset a unit's `ExecStart`, but not _why_ we want to do that.

The info the reader needs is that we re-use upstream's unit file. That is not obvious. It is implicit by `systemd.packages = [ cfg.package ];`. It should be obvious on first read.